app/gui2/src/components/GraphEditor/GraphNodes.vue

```diff
@@ -56,6 +71,7 @@ const uploadingFiles = computed<[FileName, File][]>(() => {
     @outputPortDoubleClick="emit('nodeOutputPortDoubleClick', $event)"
     @doubleClick="emit('nodeDoubleClick', id)"
     @addNode="emit('addNode', id, $event)"
+    @createNode="createConnectedNode(id, $event, node.pattern)"
```

Comment:
`node.pattern` is not necessarily an identifier. We should use getNodeFirstOutputPort to get AstId of the binding. Let's do this inside createConnectedNode and remove its last argument.